src/Psalm/Internal/Stubs/CoreGenericFunctions.php

```diff
@@ -133,7 +133,7 @@ function usort(array &$arr, callable $callback): bool {}
  * @psalm-template T
  *
  * @param array<string, T> $arr
- * @return array<string, T>
+ * @return array<string, T>|false
  */
 function array_change_key_case(array $arr, int $case = CASE_LOWER) {}
```

Comment:
I don't see where it would return `false` having passed valid arguments: https://github.com/php/php-src/blob/master/ext/standard/array.c#L4453-L4484

Reply:
yep, I think you are right ... maybe the documentation is not up-to-date https://www.php.net/manual/en/function.array-change-key-case.php ... I will revert it.